tests/Mysql5ConnectionPoolTest.php

```diff
@@ -41,8 +42,7 @@ protected function getConnection(LoopInterface $loop): Connection
             '3306',
             'root',
             'root',
-            'test',
-            [], 10
-        ), $mysqlPlatform);
+            'test'
+        ), $mysqlPlatform, new ConnectionPoolOptions(10));
```

Comment:
Could we test this new feature somehow?

Reply:
we already use the fork in our MSP Challenge project. I will do a re-test and make a quick video capture to share it as "proof"

Reply:
Let's be Brave! ;)